ood-portal-generator/templates/no-auth.conf.erb

```diff
@@ -0,0 +1,19 @@
+<VirtualHost *:<%= @port %>>
+  <%- if @servername -%>
+  ServerName <%= @servername %>
+  <%- end -%>
+
+  Redirect / /public/need_auth.html
```

Comment:
Why not just put this into main template if `auth?` returns false?  That way only dealing with one template.  Could even use rewrite rules to force all URLs to this need_auth page.

Reply:
I thought it'd much easier to implement and read. We may be able to have a giant if block(s). A quick spot check shows at least 3.  Plus the public directory is reconfigurable in the original - it's not here because it's hard coded in the packaging files so there's a little more if logic to this directory.

This is the result I'd like though, no other URLs (indeed we should probably also have 404 automatic redirects to `need_auth.html`).  I was also thinking for security it'd be better just to have  another template so we don't accidentally leak something.  We've got a lot of test coverage, but this could probably add some gaps.

Reply:
@johrstrom Try replacing `Redirect` with this:

```
RewriteEngine On
RewriteCond %{REQUEST_URI} !/public/need_auth.html
RewriteRule ^.*$ /public/need_auth.html [R=503,L]
ErrorDocument 503 /public/need_auth.html
```

This mirrors the logic we have in place for maintenance mode.

Reply:
Thanks!